test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Please, add `test.md` file inside the `.gitignore` file after creating it, the `.gitignore` file is super important in cases we don't want to push some files such as in our case `test.md` file by mistake ('Don't worry, everybody makes mistakes)